webui/react/src/components/LogViewer/LogViewerFilters.test.tsx

```diff
@@ -66,21 +66,20 @@ describe('LogViewerFilter', () => {
     });
   });
 
-  it('should render filters with rank 0 and no rank', async () => {
+  it('should render filters with rank 0 and 1', async () => {
     const values: Filters = {
       agentIds: [],
       allocationIds: [],
       containerIds: [],
       levels: [],
-      rankIds: [0, undefined],
+      rankIds: [0, 1],
     };
     const { user } = setup(values, { ...values, rankIds: [] });
 
     const agentOption1 = screen.getByText('All Ranks');
     await user.click(agentOption1);
     await waitFor(async () => {
       expect(await screen.findAllByText('0')).toHaveLength(2);
-      expect(screen.queryByText('No Rank')).toBeInTheDocument();
```

Comment:
We should have at least one test of No Rank

Reply:
there is in the bottom